exampleSite/content/blog/llm-privacy-plaigarism.md

```diff
@@ -0,0 +1,112 @@
+
+All of the above solutions have pitfalls; none offer complete guarantees of privacy and copyright protection. And a critical legal question remains unresolved: what responsibility do model providers have to ensure their products don't violate privacy and intellectual property rights? This will undoubtedly be an active area of discussion and litigation in the coming years. 
+
+
+## References
+
+Brown, H., Lee, K., Mireshghallah, F., Shokri, R., & Tramèr, F. (2022). What Does it Mean for a Language Model to Preserve Privacy? 2022 ACM Conference on Fairness, Accountability, and Transparency, 2280–2292. [link](https://dl.acm.org/doi/abs/10.1145/3531146.3534642)
+
+Carlini, N., Tramèr, F., Wallace, E., Jagielski, M., Herbert-Voss, A., Lee, K., Roberts, A., Brown, T., Song, D., Erlingsson, Ú., Oprea, A., & Raffel, C. (2021). Extracting training data from large language models. 30th USENIX Security Symposium (USENIX Security 21), 2633–2650. [link](https://www.usenix.org/system/files/sec21-carlini-extracting.pdf)
+
+Inan, H. A., Ramadan, O., Wutschitz, L., Jones, D., Rühle, V., Withers, J., & Sim, R. (2021). Training Data Leakage Analysis in Language Models. In arXiv [cs.CR]. arXiv. [link](http://arxiv.org/abs/2101.05405)
+
+Kandpal, N., Wallace, E., & Raffel, C. (2022). Deduplicating Training Data Mitigates Privacy Risks in Language Models. In arXiv [cs.CR]. arXiv. [link](http://arxiv.org/abs/2202.06539)
+
+Lee, K., Ippolito, D., Nystrom, A., Zhang, C., Eck, D., Callison-Burch, C., & Carlini, N. (2022). Deduplicating Training Data Makes Language Models Better. Proceedings of the 60th Annual Meeting of the Association for Computational Linguistics (Volume 1: Long Papers), 8424–8445. [link](https://aclanthology.org/2022.acl-long.577.pdf)
+
+Perez, E., Huang, S., Song, F., Cai, T., Ring, R., Aslanides, J., Glaese, A., McAleese, N., & Irving, G. (2022). Red Teaming Language Models with Language Models. In arXiv [cs.CL]. arXiv. [link](http://arxiv.org/abs/2202.03286)
+
+
+---
+An earlier version of this post was published at [www.carol-anderson.com](https://www.carol-anderson.com/blog/large-language-models-can-steal-work-and-spill-secrets-should-we-care). 
+
+## About the Author
+[Carol Anderson](https://www.linkedin.com/in/carolmanderson/) is a data scientist and machine learning practitioner with expertise in natural language processing (NLP), biological data, and AI ethics. She previously worked as a data scientist at [NVIDIA](https://www.nvidia.com/en-us/), [ConcertAI](https://www.concertai.com), and [Ancestry](https://www.ancestry.com/), and prior to that as an academic molecular biologist at [UCSF](https://www.ancestry.com/) and [Indiana University](https://www.indiana.edu). Currently, her main interest is operationalizing AI ethics and safety, especially in the area of generative AI.
```

Comment:
The UCSF link seems to be broken for this and previous posts. Apologies for not catching this in earlier reviews.

Reply:
Oops, yes - my mistake. Good catch! Just fixed them.